src/app/organizations/organization-details/manage-projects/manage-single-project/manage-payments/revenue-share/revenue-share.component.ts

```diff
@@ -40,4 +42,9 @@ export class RevenueShareComponent {
             }
         });
     }
+
+    backToEditProjectScreen () {
+        this.router.navigate([`/dash/manage_groups/${this.orgID}
+        /manage_project/${this.projectID}`]);
+    }
```

Comment:
Can you use `[routerLink]="['/dash', 'manage_groups', orgID, 'manage_project', projectID] | coopPath"` inside the template instead of this function? I've figured out that we might use `(click)=""` listeners too often. We can use it if we do something else along with `router.navigate()`.